src/cli/server.js

```diff
@@ -27,10 +27,14 @@ export default function startWebpack(config) {
   }
 
   if (program.production) {
-    compiler.run((err) => {
+    compiler.run((err, stats) => {
       if (err) {
         console.log(err);
       }
+      if (stats.hasErrors() || stats.hasWarnings()) {
+        console.log(stats.toJson().errors);
```

Comment:
I guess we use `console.log` elswhere, so might be fine to just use that + chalk